src/python/WMComponent/JobSubmitter/JobSubmitterPoller.py

```diff
@@ -740,9 +750,9 @@ def algorithm(self, parameters=None):
         if self.useReqMgrForCompletionCheck:
             # only runs when reqmgr is used (not Tier0)
             self.removeAbortedForceCompletedWorkflowFromCache()
-            agentConfig = self.reqAuxDB.getWMAgentConfig(self.config.Agent.hostName)
-            self.condorFraction = agentConfig.get('CondorJobsFraction', 0.75)
-            self.condorOverflowFraction = agentConfig.get("CondorOverflowFraction", 0.2)
+            self.agentConfig = self.reqAuxDB.getWMAgentConfig(self.config.Agent.hostName)
+            self.condorFraction = self.agentConfig.get('CondorJobsFraction', 0.75)
```

Comment:
how about making agentConfig a local variable and then
`self.enableAllSites = agentConfig.get("SpeedDrainConfig")['EnableAllSites']['Enabled']`
?

If so, then the code above has to be updated and in the __init__ method we should add
`self.enableAllSites = False`